bedrock/foundation/templates/foundation/annualreport/2020/index.html

```diff
@@ -0,0 +1,858 @@
+    <p>Mozilla isn’t the only one operating in this space. Citizens and communities around the globe are also making AI transparency a reality. At NYU, Laura Edelson and the AdObserver team are <a href="https://medium.com/cybersecurity-for-democracy/testimony-of-laura-edelson-nyu-cybersecurity-for-democracy-e0b7e046eb8" target="_blank" rel="external noopener">standing up to Facebook</a> about opaque political ads. And the Markup is doing similar, vital work with <a href="https://themarkup.org/series/citizen-browser" target="_blank" rel="external noopener">Citizen Browser</a>.</p>
+    <p>In 2021, we also saw citizens and communities push for better data stewardship. By giving people, and not platforms, more control over data, we can rebalance the power dynamic online. The concept of Uber or Facebook controlling all our data — with little or no benefit to society at large — is falling out of favor.</p>
+    <p>Meanwhile, organizations like <a href="https://www.theguardian.com/technology/2021/feb/19/uber-drivers-workers-uk-supreme-court-rules-rights" target="_blank" rel="external noopener">Worker Info Exchange</a> in the UK are putting workers, not platforms, in control of their data. (Worker Info Exchange is also a member of our <a href="https://foundation.mozilla.org/en/data-futures-lab/" target="_blank" rel="external noopener">Data Futures Lab</a>.) And at Mozilla, products like Common Voice and Rally are making data more democratic. <a href="https://commonvoice.mozilla.org/en" target="_blank" rel="external noopener">Common Voice</a> is building an open-source data set to make voice technology more inclusive. And <a href="https://rally.mozilla.org/" target="_blank" rel="external noopener">Rally</a> allows everyday internet users to donate their data to projects they believe in.</p>
+    <p>So what does all this add up to? In 2021, the call for trustworthy AI began to feel like a genuine, robust movement — a critical mass of people making sure technology helps, rather than harms, humanity. And it’s growing to match other vital social movements of our time, like the push for racial and environmental justice. The values, energy and innovation Mozilla brings to tools like Firefox and Pocket are also being harnessed to fuel this new movement to ensure a bright future for artificial intelligence.</p>
+    <p>As the world changes, the intersections between these different movements become clearer — from the researchers <a href="https://www.ajl.org/">unmasking biased algorithms</a>, to activists holding platforms to account for their <a href="https://stopfundingheat.info/about-the-campaign/" target="_blank" rel="external noopener">impact on the environment</a>.</p>
+    <p>In 2022, Mozilla will continue the important work of growing this movement — and making trustworthy AI a reality.</p>
+  </article>
+
+ <!-- Angela & Roxi letter -->
+  <article data-modal-parent="angela-roxi-letter" class="mzp-c-article c-leadership-article" id="angela-roxi-letter">
```

Comment:
Nit: should probably update the names here.

```suggestion
  <article data-modal-parent="angela-eric-letter" class="mzp-c-article c-leadership-article" id="angela-eric-letter">
```